src/google_calendar/models.rs

```diff
@@ -0,0 +1,72 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize)]
```

Comment:
I wonder if we really need to derive `Serialize` for all these models. So far we only deserialize them from the response and do not need to serialize them anywhere, no?